deploy/deployment-manager/deploy_service_account.sh

```diff
@@ -35,7 +35,7 @@ fi
 
 result="$(gcloud deployment-manager deployments create --automatic-rollback-on-error "${DEPLOYMENT_NAME}" --project "${PROJECT_NAME}" \
     --template service_account.py \
-    --properties scope:"${SCOPE}",parentId:"${PARENT_ID}",serviceAccountName:"${SERVICE_ACCOUNT_NAME}")"
+    --properties "scope:'${SCOPE}',parentId:'${PARENT_ID}',serviceAccountName:'${SERVICE_ACCOUNT_NAME}'")"
```

Comment:
this format is required to ensure the values passed are treated as strings

see https://cloud.google.com/sdk/gcloud/reference/deployment-manager/deployments/create